src/site/content/en/blog/yahoo-japan-news-bfcache/index.md

```diff
@@ -0,0 +1,161 @@
+
+- **Unload Handlers:** Use the `pagehide` event instead of `unload` event, as the `unload` event is [very unreliable](/bfcache/#never-use-the-unload-event). Also, `permission-policy: unload` was [launched in version M115](https://chromestatus.com/feature/5579556305502208) so that the websites can reliably remove `unload` handlers for specific origins. Chrome is also planning to [gradually deprecate `unload` handlers](https://github.com/fergald/docs/blob/master/explainers/permissions-policy-deprecate-unload.md#logistics-of-deprecation).  
+  
+  You can check what major blockers there are for your website by going to **Chrome Dev Tools > Applications > Back/forward Cache** ([more details](/bfcache/#test-to-ensure-your-pages-are-cacheable)), or by calling [`notRestoredReasons` API](https://github.com/rubberyuzu/bfcache-not-retored-reason/blob/main/NotRestoredReason.md) for a more comprehensive view of blockers based on actual usage in the wild.
+- **`Cache-control: no-store`** (or CCNS for short): Changing the `Cache-control` header from **`no-store`** to **`no-cache`** can enable bfcache. Chrome is also planning to start [caching for bfcache even with a `no-store` header](https://chromestatus.com/feature/6705326844805120).
+
+CCNS is intended for pages that should never be cached, under any circumstance. This comes with an obvious caveat—the page with CCNS will not be able to benefit from any caching technology, including CDN edge servers and local caches.
+
+If you have a CCNS header, this is a great opportunity to discuss what the right `Cache-control` strategies are for your website. Here are the main differences between `no-store` and `no-cache`.
+
+<div class="table-wrapper scrollbar">
+  <table>
+    <thead>
+      <tr>
+        <th><code>Cache-control: <strong>no-store</strong></code></th>
+        <th><code>Cache-control: <strong>no-cache</strong></code></th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <ul>
+            <li>The response is not allowed to be stored in caches.</li>
+            <li>Consequently, the response is fetched in full on every request.</li>
+          </ul>
+        </td>
+        <td>
+          <ul>
+            <li>The response is allowed to be stored in caches as long as it's revalidated with the server before each use.</li>
+          </ul>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+If you are interested in learning more about `Cache-control` options, [this article](/http-cache/?hl=en#flowchart) would be a great help.
+
+## Impact of bfcache in numbers
+
+To measure the impact of bfcache, Yahoo! JAPAN News conducted an A/B test for 2 weeks, where they served a version of their pages with their bfcache fixes to one group, and a version without bfcache to another. They picked the URL paths with a significant amount of traffic so that the test could achieve meaningful results. There is no visual or functional difference between the 2 versions.
```

Comment:
Pick a tense and stick with it :-)


```suggestion
To measure the impact of bfcache, Yahoo! JAPAN News conducted an A/B test for 2 weeks, where they served a version of their pages with their bfcache fixes to one group, and a version without bfcache to another. They picked the URL paths with a significant amount of traffic so that the test could achieve meaningful results. There was no visual or functional difference between the 2 versions.
```